packages/0x.js/src/contract_wrappers/ether_token_wrapper.ts

```diff
@@ -163,6 +163,15 @@ export class EtherTokenWrapper extends ContractWrapper {
     public unsubscribeAll(): void {
         super.unsubscribeAll();
     }
+    /**
+     * Retrieves the Ethereum address of the EtherToken contract deployed on the network
+     * that the user-passed web3 provider is connected to. If it's an unknown private network - undefined is returned.
```

Comment:
How about instead of `an unknown private network`, we said `it's not Kovan, Ropsten, Rinkeby, Mainnet or TestRPC (networkId: 50), it will return undefined (e.g a private network).`. This is more explicit.